lib/src/core/util/constants.dart

```diff
@@ -52,10 +52,17 @@ abstract class Constants {
   static const double cardInfoContainerSpreadRadius = 5;
   static const double cardInfoContainerBlurRadius = 5;
   static const bool titleIsTrue = true;
+  static const bool titleIsFalse = false;
```

Comment:
this two constants could be combined into one, "isTitle" defaulted as true, and negate the value when you need it to be false, like this: 
isTitle == true
!isTitle == false

Reply:
Done